acceptance-tests/build.sbt

```diff
@@ -20,4 +20,4 @@ name := "acceptance-tests"
 
 parallelExecution in Test := false
 
-libraryDependencies += "org.apache.jena" % "jena-fuseki-main" % "3.11.0" % Test
+libraryDependencies += "org.apache.jena" % "jena-fuseki-main" % "3.12.0" % Test
```

Comment:
did you also change the version of the jena image that is used?

Reply:
I did upgrade (and tested) the latest version of jena-fuseki image.